### PR TITLE
VertexLoader: clean up and fix some problems

### PR DESCRIPTION
**Pull request by Tilka**

### CODE REVIEW

**Comment by Tilka** [comment]
I'll change this to use the ABI functions for adjusting the stack, please don't merge until then.

**Comment by delroth** [comment]
@dolphin-emu-bot rebuild (/me preseeding a new fifoci image release)

File: Source/Core/VideoCommon/VertexLoaderManager.cpp [hunk 1]
```diff
@@ -175,8 +175,9 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	return size;
 }
 
-int GetVertexSize(int vtx_attr_group, bool preprocess)
+int GetVertexSize(int vtx_attr_group)
 {
+	bool preprocess = true;
```
Comment (degasus) [inline Source/Core/VideoCommon/VertexLoaderManager.cpp]: Why? I'm not even sure this is safe. Without deterministic dual core, this maybe won't get updated at all.

Comment (Tilka) [inline Source/Core/VideoCommon/VertexLoaderManager.cpp]: Removed it from this PR.

**Pull request merged**